packages/features/ee/teams/lib/payments.ts

```diff
@@ -117,10 +117,8 @@ export const updateQuantitySubscriptionFromStripe = async (teamId: number) => {
       return;
     }
 
-    const newQuantity = membershipCount - subscriptionQuantity;
-
```

Comment:
We fetch team.members earlier here, my guess this function was being called before adding team member. But now as we fetch team.member and we already count the latest added we can update quantity to membershipCount.